src/customer/customer-request-options.ts

```diff
@@ -101,3 +108,17 @@ export interface CustomerInitializeOptions extends CustomerRequestOptions {
      */
     googlepaystripe?: GooglePayCustomerInitializeOptions;
 }
+
+/**
+ * A set of options that are required to pass the customer step of the
+ * current checkout flow.
+ *
+ * Some payment methods have specific suggestion for customer to pass
+ * the customer step. For example, Bolt suggests the customer to use
+ * their custom checkout with prefilled form values. As a result, you
+ * may need to provide additional information.
+ *
+ */
+export interface ExecutePaymentMethodCheckoutOptions extends CustomerRequestOptions {
+    fallback?(): void;
```

Comment:
This method seems to be doing a few things, and fallback is a very generic name. On discussion this feels like something more like `continueWithCheckoutCallback` ?

Reply:
`continueWithCheckoutCallback` is more valuable and matching name for this case. Thanks, I will fix it today